metadata-ingestion/src/datahub/emitter/mcp.py

```diff
@@ -98,6 +99,9 @@ def __post_init__(self) -> None:
                 f"aspectName {self.aspectName} does not match aspect type {type(self.aspect)} with name {self.aspect.get_aspect_name()}"
             )
 
+        if not self.systemMetadata:
+            self.systemMetadata = SystemMetadataClass(lastObserved=get_sys_time())
```

Comment:
not so sure about this change

Reply:
We need to somehow "stamp" our metadata at the time of creation of MCPW object instead of when transformer logic finishes - to avoid having stamp time being to much in future compared with the moment it was scraped from the source system - this creates confusion.
This means we have to completely change approach to the timestamping and move it closer to the scraping moment.